src/main/resources/liquibase/crop_changelog/21_4_0.xml

```diff
@@ -0,0 +1,144 @@
+				VALUES (8341, 1010, 'Female Parent', 'Female Parent', 0, 0, 0);
+
+				INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+				VALUES (1225, 8341, 1087);
+			</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.4.0-2">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'Male Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8344;</sqlCheck>
+				</and>
+			</preConditions>
+			<comment>Add 'Male Parent' property</comment>
+			<sql dbms="mysql" splitStatements="true">
+				INSERT INTO CVTERM (cvterm_id, cv_id, name, definition, is_obsolete, is_relationshiptype, is_system)
+				VALUES (8344, 1010, 'Male Parent', 'Male Parent', 0, 0, 0);
+
+				INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+				VALUES (1225, 8344, 1087);
+			</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.4.0-3">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'FEMALE_PARENT_NAME' and cv_id = 1040 and is_obsolete = 0;</sqlCheck>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where cvterm_id = 8236;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm where name = 'Female Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8341;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm_relationship where type_id = 1225 and subject_id = 8341 and object_id = 1087;</sqlCheck>
+
+				</and>
+			</preConditions>
+			<comment>Add 'FEMALE PARENT NAME' as germplasm descriptor</comment>
+			<sql dbms="mysql" splitStatements="true">
+				INSERT INTO cvterm (cvterm_id, cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype, is_system)
+				VALUES (8236, 1040, 'FEMALE_PARENT_NAME','Name of Female Parent', NULL, 0, 0, 1);
+
+				INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+				VALUES
+					(1200, 8236, 8341),
+					(1210, 8236, 4030),
+					(1220, 8236, 1908);
+
+				INSERT INTO cvtermprop (cvterm_id, type_id, value, rank) VALUES
+					(8236, 1800, 'Germplasm Descriptor', 0),
+					(8236, 8048, (SELECT DATE_FORMAT(CURRENT_TIMESTAMP(), '%Y-%m-%dT%T.000Z')), 0);
+			</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.4.0-4">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'FEMALE_PARENT_GID' and cv_id = 1040 and is_obsolete = 0;</sqlCheck>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where cvterm_id = 8237;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm where name = 'Female Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8341;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm_relationship where type_id = 1225 and subject_id = 8341 and object_id = 1087;</sqlCheck>
```

Comment:
@cuenyad same here, maybe this is not needed

Reply:
done!